internal/server/tests.go

```diff
@@ -452,6 +452,7 @@ func grpcHealthCheckPasses(grpcConn *grpc.ClientConn, reqTimeout time.Duration)
 
 		resp, err := client.Check(ctx, &healthpb.HealthCheckRequest{})
 		if err != nil {
+			t.Errorf("gRPC health check failed: %s", err.Error())
```

Comment:
Errorf actually fails the test.

```suggestion
			t.Logf("gRPC health check failed: %v", err)
```